### ISSUE TITLE
API/Example plugin?

### ISSUE THREAD

**Issue by ROTNdev** [WorldCretornica/PlotMe-Core#128]
I've been trying to convert a plugin that depended on the old PlotMe to PlotMe-Core with no luck. I'm using the current release as the maven dependency, but it doesn't let me access any of the static methods in PlotMeCoreManager.

I only need to get the plot the player is in, and to see if the player is able to build in it.

Help is appreciated, thanks.

**Comment by ZachBora** [comment]
I'm updating the Example plugin

**Comment by ZachBora** [comment]
@ROTNdev I've updated PlotMeHookExample.

https://github.com/WorldCretornica/PlotMeHookExample/blob/PlotMeCore/src/main/java/com/worldcretornica/plotmehookexample/PlotMeHookExample.java

Notes:
* Made using v0.16, but most of the code should work for 0.15.x
* I noticed some functions are static and others are not, this is not final and we might remove the statics or move them for easier use.
* The function to get the number of plots was deprecated, I did not know we forgot to make it, so I fixed that just now in PlotMe.
